components/authz-service/storage/postgres/migration/sql/71_sync_query_projects_with_query_project.up.sql

```diff
@@ -0,0 +1,39 @@
+BEGIN;
+
+CREATE OR REPLACE FUNCTION
+  query_projects(_projects_filter TEXT[])
+  RETURNS setof json AS $$
+
+WITH
+staged AS (
+  SELECT r.project_id
```

Comment:
This line (and L15 similarly) previously read
```
  SELECT r.project_id, count(r.id) AS count
```
The `count` was actually not needed (thanks @blakestier !) so was removed when `query_project` (singular) was created; just mirroring that change here.